.github/workflows/integration.yml

```diff
@@ -12,7 +12,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        test-suite: [TestEthereumE2ESuite, TestFabricE2ESuite]
+        test-suite: [TestEthereumMultipartyE2ESuite, TestEthereumGatewayE2ESuite, TestEthereumMultipartyTokensRemoteNameE2ESuite, TestEthereumNamespaceE2ESuite, TestFabricGatewayE2ESuite, TestFabricMultipartyE2ESuite]
```

Comment:
Should we include the migration tests?

Reply:
There's already a separate job (line 60)